src/components/themed/FlipInput.js

```diff
@@ -567,4 +604,7 @@ const getStyles = cacheStyles((theme: Theme) => ({
   }
 }))
 
-export const FlipInput = withTheme(FlipInputComponent)
+const FlipInputThemed = withTheme(FlipInputComponent)
+
+// $FlowFixMe - forwardRef is not recognize by flow?
```

Comment:
Yeah, that makes sense. Flow doesn't know about this function, which was introduced around the same time as hooks. We need like Flow 0.89 or higher to get it, which is a huge breaking change, so the plan is to jump straight to Typescript instead.